# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/StaticPrefList.yaml
# Commit: 89e396b2896e
# Full Hash: 89e396b2896ed7fc168a5b4ec2f8ead05f5c6802
# Author: Andrea Marchesini <amarchesini@mozilla.com>
# Date: 2020-06-03 04:20:58
# Regressor Bug: 1638358
# File Overlap Count: 1
# Description:
#   Bug 1638358 - Cookie Schemeful Same-Site - part 5 - schemeful comparison, r=mayhemer
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D75629
# ==============================================================================

diff -r 33ad5fa05209 -r 89e396b2896e modules/libpref/init/StaticPrefList.yaml
--- a/modules/libpref/init/StaticPrefList.yaml	Tue Jun 02 09:14:09 2020 +0000
+++ b/modules/libpref/init/StaticPrefList.yaml	Tue Jun 02 09:15:24 2020 +0000
@@ -7505,6 +7505,11 @@
   value: @IS_NIGHTLY_BUILD@
   mirror: always
 
+- name: network.cookie.sameSite.schemeful
+  type: bool
+  value: @IS_NIGHTLY_BUILD@
+  mirror: always
+
 - name: network.cookie.thirdparty.sessionOnly
   type: bool
   value: false